boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -332,3 +333,17 @@ opm-build-push: docker-push
 .PHONY: ensure-fips
 ensure-fips:
 	${CONVENTION_DIR}/configure-fips.sh
+
+# You will need to export the forked/cloned operator repository directory as OLD_SDK_REPO_DIR to make this work.
+# Example: export OLD_SDK_REPO_DIR=~/Projects/My-Operator-Fork
+.PHONY: migrate-to-osdk1
+migrate-to-osdk1:
+ifndef OLD_SDK_REPO_DIR
+$(error OLD_SDK_REPO_DIR is not set)
```

Comment:
Indent just this line.

As written, this will include the error line at the base level of the file rather than as part of the rule, so you would need to have the variable set always. (It's the same as what's happening on L1-16.)

Whereas if you indent this line, you're affecting how the *rule* gets "compiled".

So as
```suggestion
	$(error OLD_SDK_REPO_DIR is not set)
```
...you can think of it this way: With `OLD_SDK_REPO_DIR` *set* your rule becomes:
```make
.PHONY: migrate-to-osdk1
migrate-to-osdk1:
	$(error OLD_SDK_REPO_DIR is not set)
	# Copying files & folders
	rm -rf config
	rsync -a $(OLD_SDK_REPO_DIR)/deploy . --exclude=crds
	rsync -a $(OLD_SDK_REPO_DIR)/pkg . --exclude={'apis','controller'}
	rsync -a $(OLD_SDK_REPO_DIR)/.gitignore .
	rsync -a $(OLD_SDK_REPO_DIR)/ . --exclude={'cmd','version','boilerplate','deploy','pkg'} --ignore-existing
```
...and of course never gets past that first line *when the rule is invoked*.
Whereas with it *unset* the rule becomes:
```make
.PHONY: migrate-to-osdk1
migrate-to-osdk1:
	# Copying files & folders
	rm -rf config
	rsync -a $(OLD_SDK_REPO_DIR)/deploy . --exclude=crds
	rsync -a $(OLD_SDK_REPO_DIR)/pkg . --exclude={'apis','controller'}
	rsync -a $(OLD_SDK_REPO_DIR)/.gitignore .
	rsync -a $(OLD_SDK_REPO_DIR)/ . --exclude={'cmd','version','boilerplate','deploy','pkg'} --ignore-existing
```

Reply:
I knew something was off... (:facepalm:) thanks for clarifying.